src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -1088,6 +1149,11 @@ def __init__(self, config: Wav2Vec2Config):
         else:
             self.encoder = Wav2Vec2Encoder(config)
 
+        if config.add_adaptor:
+            self.adaptor = Wav2Vec2Adaptor(config)
+        else:
+            self.adaptor = None
```

Comment:
Short enough to go on one line.
```suggestion
        self.adaptor = Wav2Vec2Adaptor(config) if config.add_adaptor else None
```